.github/workflows/jira.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Create ticket
         if: ( github.event.action == 'opened' && steps.set-ticket-type.outputs.type != 'Task' ) || ( github.event.action == 'opened' && steps.set-ticket-type.outputs.type == 'Task' )
-        uses: tomhjp/gh-action-jira-create@v0.1.3
+        uses: tomhjp/gh-action-jira-create@v0.2.0
```

Comment:
Going back to 2.0?

Reply:
Yea I don't think the 0.1.3 made any difference, the issue was Auth was not working when using the action when custom fields were used in JIRA